warp10/src/main/java/io/warp10/standalone/StandaloneDeleteHandler.java

```diff
@@ -214,6 +214,11 @@ public void handle(String target, Request baseRequest, HttpServletRequest reques
 
     String selector = request.getParameter(Constants.HTTP_PARAM_SELECTOR);
 
+    if (null == selector) {
+      response.sendError(HttpServletResponse.SC_BAD_REQUEST, "Invalid '" + Constants.HTTP_PARAM_SELECTOR +"' query parameter, should not be empty.");
```

Comment:
Isn't missing, instead of empty ?.
Or if you change the condition to null == selector || selector.isEmpty() , you can return "missing or empty selector query parameter".

Reply:
It makes sense to handle both case as this field is needed to be fullfilled. Pushed the changes.